pkg/imgpkg/cmd/push.go

```diff
@@ -92,21 +95,60 @@ func (po *PushOptions) Run() error {
 		panic("Unreachable code")
 	}
 
-	po.ui.BeginLinef("Pushed '%s'", imageURL)
+	po.ui.BeginLinef("\nPushed: \n%s\n", imageURL)
 
 	return nil
 }
 
 func (po *PushOptions) pushBundle(registry registry.Registry) (string, error) {
-	uploadRef, err := regname.NewTag(po.BundleFlags.Bundle, regname.WeakValidation)
+
+	imageURL := ""
+	imageRefs := []string{}
+	uploadRefs := []regname.Tag{}
+
+	baseImageName, err := po.stripTag()
+	if err != nil {
+		return "", err
+	}
+
+	baseRef, err := regname.NewTag(po.BundleFlags.Bundle, regname.WeakValidation)
 	if err != nil {
 		return "", fmt.Errorf("Parsing '%s': %s", po.BundleFlags.Bundle, err)
 	}
 
+	// Append the base image_tag to the list of refs to upload
+	uploadRefs = append(uploadRefs, baseRef)
```

Comment:
I think that this might be more readable if we do:
```suggestion
	imageRefs := []string{baseRef}
```

Reply:
I'm not 100% sure what the suggestion is here. Do you want to rename the variable from `uploadRefs` to `imageRefs`, or change the data structure from `[]regname.Tag` to `[]string`, or move to a single initialization and assignment, or all of the above?

Reply:
Bad copy paste :( 
This is what I meant:
```
uploadRefs := []regname.Tag{baseRef}
```

Reply:
Got it!